lib/turn.rb

```diff
@@ -1,12 +1,23 @@
 class Turn
-  attr_accessor :board, :player, :turn_number
+  attr_accessor :board, :player, :computer, :turn_number
+  
   def initialize(board, player)
     @board = board
     @player = player
-    # @computer = computer
+    @computer = computer
     @turn_number = 0
-
   end
 
+  def place_selection
+    p "Choose a column A-G to drop your piece!"
+    selection = gets.chomp.upcase
+    
+    until @board.grid.first.include?(selection)
+      p "Please try again"
+      selection = gets.chomp.upcase
+    end
```

Comment:
if user type outside of A-G, it will print "Please try again" until the user's input is in the range of A-G